style.css

```diff
@@ -352,3 +402,212 @@ form ::placeholder {
   font-size: 15px;
   line-height: 24px;
 }
+
+.menu ul {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .homepage {
+    background-image: url(images/Header\ bg.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .logo {
+    font-size: 18px;
+    line-height: 20px;
+    color: #6070ff;
+    font-weight: 700;
+    text-decoration: none;
+    padding-left: 144px;
+  }
+
+  .main_container_one {
+    width: 100%;
+    padding-top: 240px;
+    padding-left: 240px;
+    padding-right: 240px;
+  }
+
+  .main_container_one h1 {
+    width: 310px;
+  }
+
+  .menu img {
+    display: none;
+  }
+
+  .menu ul {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 30px;
+    padding-right: 154px;
+    text-decoration: none;
+  }
+
+  .menu li {
+    text-decoration: none;
+    list-style: none;
+  }
+
+  .menu a {
+    font-size: 15px;
+    font-weight: 500;
+    line-height: 20px;
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .work_section {
+    padding-left: 142px;
+    padding-right: 142px;
+  }
+
+  .card_container {
+    width: 100%;
+    display: grid;
+    gap: 88px;
+  }
+
+  .card_one {
+    width: 100%;
+    background-color: #fff;
+    border-radius: 16px;
+    display: flex;
+    border: 1px solid #dfe1e6;
+    padding: 20px;
+    gap: 28px;
+  }
+
+  .card_one:nth-child(even) {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .card_image img {
+    width: 544px;
+  }
```

Comment:
You did a brilliant job in here. But let's make the works card responsive and when they respond to the different size, they still be look like on your desktop version. Half of the card is image and other half is text.
<img width="1036" alt="Screenshot 2023-01-19 at 10 06 40 PM" src="https://user-images.githubusercontent.com/25887253/213540017-04a4a779-878b-4e0c-9433-c3ac4c0dadcc.png">